Update README.md (#26)
ocp-power-automation/ocp4-upi-multiarch-compute: tf/add-powervm-workers/README.md

> 
> 

Reviewer: ```suggestion
```

## Troubleshooting

> 
> 

Reviewer: ```suggestion

```

> ### If Ignition failed.

Reviewer: ```suggestion
### Ignition failed.
```

* If ignition failed, Once you have the VM console, run the following commands:
```
systemctl --failed
journalctl -u ignition-fetch
```
2. Ignition happens in a couple of steps
```
ignition -platform openstack -stage mount -log-to-stdout
```
then fetch
```
ignition --platform openstack -stage fetch -log-to-stdout
```
```
sudo journalctl -u kubelet
```
* Ignition file properties should has `/etc/hostname` , `/etc/resolv.conf`, `passwd` section with proper values.

> 
> 
> 

Reviewer: ```suggestion

```

> ### Even after the ignition was sucessful , CSR didn't appear 

Reviewer: ```suggestion
### Ignition successful, CSR does not show
```

* If you see the error with `no such host` error in the `sudo journalctl -u kubelet` add the DNS service points to the right host/ip of the ignition IP.

> * Using the dns service [link](https://cloud.ibm.com/internet-svcs/crn%3Av1%3Abluemix%3Apublic%3Ainternet-svcs%3Aglobal%3Aa%2F65b64c1f1c29460e8c2e4bbfbd893c2c%3A60ee139c-1c18-4728-a035-e34533b78a8b%3A%3A?paneId=reliability), Add a new record to resolve the IP which points to domain.

Reviewer: ```suggestion
* Using the IBM Cloud Internet Service or IBM Cloud DNS Service [link](https://cloud.ibm.com/internet-svcs/), add a new record to resolve the IP which points to domain.
```
